tests/test_queries.py

```diff
@@ -728,6 +728,80 @@ def test_stats(client: FlaskClient):
     qmcall(client, {"q": "hvaða fólk er mest í fréttum"}, "Stats")
 
 
+def test_sunposition(client: FlaskClient):
```

Comment:
If the query module is called sun_position, the test function should be test_sun_position for consistency.

Reply:
Renamed to test_sunpos along with the module itself (now called sunpos)